test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -81,11 +81,22 @@ public static TheoryData MultiAttributeCheckBoxData
             string expectedAttributeString)
         {
             // Arrange
-            var originalContent = "original content";
```

Comment:
The original test has `PreContent` (originalContent) and `Content` (expectedAttributeString) https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/InputTagHelperTest.cs#L86 so I added those. And `PostElement` (the hidden element) is now added for all checkbox type input tag helpers so I added a check for that. I can remove `PostContent` though.